docs/general-usage/middleware.rst

```diff
@@ -18,7 +18,7 @@ GrappleMiddleware
     # settings.py
     GRAPHENE = {
         # ...
-        'MIDDLEWARE': 'grapple.middleware.GrappleMiddleware',
+        'MIDDLEWARE': ['grapple.middleware.GrappleMiddleware'],
```

Comment:
Since we're using black for formatting

```suggestion
        "MIDDLEWARE": ["grapple.middleware.GrappleMiddleware"],
```

other than that, LGTM

Reply:
I lazily edited from github directly...I will avoid this in the future 🙄 
Thanks